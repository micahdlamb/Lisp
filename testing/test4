(LESS 5 6)
(LESS 5 5)
(LESS 5 4)
(GREATER 5 6)
(GREATER 5 5)
(GREATER 5 4)
(EQ 5 6)
(EQ 5 5)
(EQ 5 4)
(INT (quote 4))
(INT (QUOTE A))
(INT (quote (a . b)))
(ATOM (quote 4))
(ATOM (QUOTE A))
(ATOM (QUOTE (A . B)))
(ATOM (PLUS 4 6))
(ATOM (CONS (quote a) (PLUS 4 6)))
