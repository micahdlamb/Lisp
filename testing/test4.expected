T
NIL
NIL
NIL
NIL
T
NIL
T
NIL
T
NIL
NIL
T
T
NIL
T
NIL
